canonical-web-and-design/ubuntu.com
# Build new /kubernetes/what-is-kubernetes page

## Done

- Build new /kubernetes/what-is-kubernetes page
- Added it to the navigation
- **NOTE** - needs update of top diagram

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/what-is-kubernetes page
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [design](#7551) and the [copy doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)


## Issue / Card

Fixes #7552
